libs/common-services/src/main/java/uk/gov/di/ipv/core/library/helpers/RequestHelper.java

```diff
@@ -263,27 +246,19 @@ private static void validateIpvSessionId(
         }
     }
 
-    private static String getIpAddress(Map<String, String> headers)
-            throws HttpResponseExceptionWithErrorBody {
+    private static String getIpAddress(Map<String, String> headers) {
         String ipAddress = RequestHelper.getHeaderByKey(headers, IP_ADDRESS_HEADER);
-        validateIpAddress(ipAddress, String.format("%s not present in header", IP_ADDRESS_HEADER));
+        validateIpAddress(ipAddress);
         return ipAddress;
     }
 
-    private static void validateIpAddress(String ipAddress, String errorMessage)
-            throws HttpResponseExceptionWithErrorBody {
+    private static void validateIpAddress(String ipAddress) {
         if (ipAddress == null) {
-            LOGGER.error(LogHelper.buildErrorMessage(errorMessage, IP_ADDRESS_HEADER));
-            throw new HttpResponseExceptionWithErrorBody(
-                    SC_BAD_REQUEST, ErrorResponse.MISSING_IP_ADDRESS);
+            LOGGER.warn(LogHelper.buildErrorMessage(ErrorResponse.MISSING_IP_ADDRESS));
```

Comment:
Similarly - I'm not convinced a blank/missing IP Address should be fatal (I think it's only used for audit logging)